images/airflow-managed-services/Dockerfile

```diff
@@ -17,20 +18,17 @@ RUN curl -L $(curl -s https://api.github.com/repos/openshift-online/ocm-cli/rele
 RUN chmod +x /usr/local/bin/rosa && chmod +x /usr/local/bin/ocm
 RUN /usr/local/bin/rosa download openshift-client
 RUN tar xzvf openshift-client-linux.tar.gz
-RUN curl "https://www.python.org/ftp/python/3.9.9/Python-3.9.9.tar.xz" | tar xJf -
-RUN cd Python-3.9.9 && ./configure --enable-optimizations && make && make altinstall
+RUN mv oc kubectl /usr/local/bin/
 RUN echo "export CLOUDSDK_PYTHON=python3.9" >> /home/airflow/.bashrc
 RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - && apt-get update -y && apt-get install google-cloud-sdk -y
-RUN mv oc kubectl /usr/local/bin/
 RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
 RUN unzip awscliv2.zip
 RUN ./aws/install
 RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
 RUN echo "airflow ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
-RUN git clone --branch main https://github.com/openshift/hypershift
-WORKDIR hypershift
-RUN make build
+COPY --from=hypershift /go/hypershift/bin/hypershift /go/hypershift/bin/hypershift-operator /go/hypershift/bin/control-plane-operator /usr/local/bin/
+RUN cd /usr/local/bin && ln -s control-plane-operator ignition-server && ln -s control-plane-operator konnectivity-socks5-proxy && ln -s control-plane-operator availability-prober && ln -s control-plane-operator token-minter
```

Comment:
@morenod At the moment, install.sh script only uses `hypershift` command. so we really wont be needing `hypershift-operator` and `control-plane-operator` binaries.

Reply:
No problem, it do not consume too much space, and you have it available for using in the future if you need it